spec.md

```diff
@@ -100,8 +100,19 @@ This header allows brokers to reject requests from marketplaces for versions the
 
 ## Authentication
 
-The marketplace MUST authenticate with the service broker using HTTP
-basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
+While the communication between a platform and broker
+MAY be unsecure, it is RECOMMENDED that all communications between a
+platform and a broker are secured via TLS and authenticated.
+
+Unless there is some out of band communication and agreement between a
+platform and a service broker, the marketplace MUST authenticate with the
+service broker using HTTP basic authentication (the `Authorization:` header)
```

Comment:
the authorization header and basic auth aren't the same thing, contrary to what this seems to imply.  Seems like this should say "the marketplace must authenticate w/ the service broker using the authorization header.  The value of that header MAY be basic auth" (or it may be something else, which i think is what we're looking for here).